### ISSUE TITLE
FactorioLab doesn't seem to take water heat capacity into account

### ISSUE THREAD

**Issue by stevenwdv** [factoriolab/factorio-lab#712]
**Describe the bug**
FactorioLab doesn't seem to take water heat capacity into account when calculating how much energy it takes to heat water to produce steam. This manifests itself in incorrect steam per second for boilers with mods that change the water heat capacity, such as Krastorio 2, which changes it from 200 to 500 J/℃.

**To Reproduce**
1. [Enable Krastorio 2 and produce steam from 1 boiler](https://factoriolab.github.io/list?p=steam*1*3&i=coal*1_water*1&s=kr2*0&v=1)
2. You will see that FactorioLab says you can produce 50 steam/sec this way.

**Expected behavior**
This should be 20 steam/sec. Compare:

Incorrect heat capacity: `1.5*10^6 W / ((165℃ - 15℃) * 200 J/℃) = 50 u/sec`<br/>
Correct heat capacity: `1.5*10^6 W / ((165℃ - 15℃) * 500 J/℃) = 20 u/sec`

Note that this uses Krastorio 2's boiler energy consumption of 1.5 MW.

**Comment by dcbroad3** [comment]
You are correct, IIRC the steam recipe is essentially hard coded by the data generator app. 
The other thing the calculator doesn't handle is different temperature steam and which types of boilers can produce different temperatures. I suppose I would likely try to solve both problems when I address this, though I don't know how complex that will be.

**Comment by stevenwdv** [comment]
Ok! Took a while to figure out why the values didn't match... At least KR2 does not change the temperature, but yeah, that should also be taken into account.
